Source/Windows/ARM64EC/Module.S

```diff
@@ -53,6 +53,11 @@ BeginSimulation:
   // Expects the target code address in x9
 .global ExitFunctionEC
 ExitFunctionEC:
+  // Clear any AFP bits set in FPCR as native code won't expect their behaviour.
+  mrs x17, fpcr
+  and x17, x17, #-5 // NEP + AH + FIZ
```

Comment:
Two questions.

1. For the bits that we preserve, will they ever vary? can we just hardcode the fpcr to restore here and save an instruction?
2. should this be predicated on the hw supporting AFP? (current Apple does not, for ex)

Reply:
Since it's just clearing bits HW support shouldn't matter, and we want the rest of the bits to be preserved in arm64ec code

Reply:
Sounds good.